recipe/meta.yaml

```diff
@@ -12,16 +12,18 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
   entry_points:
     - pycc = numba.pycc:main
     - numba = numba.misc.numba_entry:main
   script:
+    - export CC="${CC} -pthread"  # [linux]
```

Comment:
Seems the previous approach didn't work. So trying to do the same thing as was done in PR ( https://github.com/numba/numba/pull/7312 ).